src/obs-plugin/plugin-main.mm

```diff
@@ -28,10 +28,14 @@
     return obs_module_text("macOS Virtual Webcam");
 }
 
+static void *data = &data;
+
 static void *virtualcam_output_create(obs_data_t *settings, obs_output_t *output)
 {
     blog(LOG_DEBUG, "output_create");
     sMachServer = [[MachServer alloc] init];
+    // `obs_output_info.create` should return non `NULL` pointer.
+    return data;
```

Comment:
I wonder what other OBS plugins do? Maybe just `return output`?

Reply:
Other plugin that is using `data` is return its own heap pointer for own `struct`.
For example, https://github.com/Palakis/obs-ndi/blob/master/src/obs-ndi-output.cpp#L247.

Reply:
Ooh yeah. Would be cleaner to store all my stuff in a struct instead of making it global/static like I do currently. In practice I don't think it makes a difference. Would you mind adding a comment above the `static void *data = &data;` that says something like `// This is a dummy pointer so we have something to return from virtualcam_output_create`?

Reply:
Yeah, that would be better comment. Let me update. And I agree that it is better for using `data` instead of global `sMachServer`, but... probably in separate patch?

Reply:
Yeah definitely fine to refactor my data storage in a separate patch. This is a solid improvement for now!